src/test/java/org/opentripplanner/middleware/controllers/api/OtpUserControllerTest.java

```diff
@@ -60,69 +61,70 @@ public static void tearDown() {
     }
 
     /**
-     * Test to check that a phone verification SMS request updates OtpUser.pendingPhoneNumber
-     * and leaves OtpUser.phoneNumber intact.
+     * Check that if for some reason the verification SMS is not sent (invalid format or other error from Twilio),
+     * the request is 400-bad request/500-error, and that the user phone number is unchanged.
      */
-    @Test
-    public void smsRequestShouldSetPendingPhoneNumberOnly() {
-        final String PHONE_NUMBER_TO_VERIFY = "+15555550321";
-        final String PHONE_NUMBER_TO_VERIFY_FORMATTED = "(555) 555-0321";
+    @ParameterizedTest
+    @MethodSource("createBadPhoneNumbers")
+    public void invalidNumbersShouldProduceBadRequest(Map.Entry<String, Integer> testCase) {
+        assumeTrue(testsShouldRun());
```

Comment:
I feel like there is a lot of churn happening on this auth disabled stuff, but I'm pretty sure in some PR we started to use the pattern of not checking `DISABLE_AUTH=false` but rather setting it with Auth0Connection#setAuthDisabled and reverting the value to getDefaultAuthDisabled in tearDown. Please follow that pattern here (it's still OK to check e2e = true).

Reply:
Good point. e2e is not required.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/83/commits/8ec525aabdaac519033d9918bca89ad483ecaa7f.